stylesheets/animate_name.css

```diff
@@ -0,0 +1,27 @@
+/* this code is based on Youtube tutorial Creative Text Animation by Online Tutorials */
+#intro__title {
+    position:relative;
+    color: var(--black);
+    -webkit-text-stroke: 0.3vw rgb(59, 58, 58);
```

Comment:
Just FYI: This is a non-standard property so might not be supported across all browsers (or might render differently).

It's worth keeping in mind/making sure that if the rule doesn't work it still looks good by commenting it out and taking a look.